cli/pcluster/config/mappings.py

```diff
@@ -457,6 +461,7 @@
     "type": CfnSection,
     "key": "fsx",
     "default_label": "default",
+    "dont_convert_to_NONE": True,
```

Comment:
Let's remove this one, it's very specific to the NONE case, instead we can generalize the CFN default setting approach as explained before.